dask/dataframe/io/parquet/core.py

```diff
@@ -395,7 +427,7 @@ def read_parquet(
         "ignore_metadata_file": ignore_metadata_file,
         "metadata_task_size": metadata_task_size,
         "split_row_groups": split_row_groups,
-        "chunksize=": chunksize,
+        "chunksize": chunksize,
```

Comment:
Just saw this change - this looks like there was a bug in the `chunksize` implementation that made it not actually work before, is that correct? If so, that may be further evidence that this option can be removed.